### PR TITLE
metrics: update resource manager panel 

### PR DESCRIPTION
**Pull request by HuSharp**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Ref https://github.com/pingcap/tidb/pull/42431

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):https://github.com/pingcap/tidb/pull/41624

### CODE REVIEW

**Comment by HuSharp** [comment]
@Oreoxmt @hfxsd PTAL thx!

**Comment by Oreoxmt** [comment]
@HuSharp Could you please invite a tech reviewer? Thanks!

**Comment by HuSharp** [comment]
@CabinfeverB @JmPotato PTAL, thx!

File: grafana-resource-control-dashboard.md [hunk 1]
```diff
@@ -31,6 +31,14 @@ summary: 了解资源管控 (Resource Control) 的 Grafana Dashboard 中所展
 - Bytes Written：以 Resource Group 为单位进行实时统计的写入数据量。`total` 为当前所有 Resource Group 写入数据量之和。
 - Bytes Written Per Query：平均每个 SQL 语句的写入数据量。计算方法是将前述 Bytes Written 监控指标除以当前每秒执行的 SQL 语句数量。
 - KV CPU Time：以 Resource Group 为单位进行实时统计的 KV 层 CPU 时间消耗。`total` 为当前所有 Resource Group 消耗 KV 层 CPU 时间之和。
-- KV CPU Time Per Query：平均每个 SQL 语句的 KV 层 CPU 时间消耗之和。计算方法是将前述 KV CPU Time 监控指标除以当前每秒执行的 SQL 语句数量。
 - SQL CPU Time：以 Resource Group 为单位进行实时统计的 SQL 层 CPU 时间消耗。`total` 为当前所有 Resource Group 消耗 SQL 层 CPU 时间之和。
-- SQL CPU Time Per Query：平均每个 SQL 语句的 SQL 层 CPU 时间消耗之和。计算方法是将前述 SQL CPU Time 监控除以当前每秒执行的 SQL 语句数量。
+
+## Client 相关指标
+
+- Active Resource groups：实时统计各个 Controller Client 的 Resource Groups 数量。
```
Comment (JmPotato) [inline grafana-resource-control-dashboard.md:38]:
```suggestion
- Active Resource groups：实时统计各个 Resource Control Client 的 Resource Groups 数量。
```

**Review by JmPotato — APPROVED** [review]

**Comment by HuSharp** [comment]
> @HuSharp Could you please invite a tech reviewer? Thanks!

Completed , PTAL, thx! @Oreoxmt

**Review by hfxsd — APPROVED** [review]

File: grafana-resource-control-dashboard.md [hunk 2]
```diff
@@ -22,15 +22,23 @@ summary: 了解资源管控 (Resource Control) 的 Grafana Dashboard 中所展
...
 - Bytes Read：以 Resource Group 为单位进行实时统计的读取数据量。`total` 为当前所有 Resource Group 读取数据量之和。
 - Bytes Read Per Query：平均每个 SQL 语句的读取数据量。将前述 Bytes Read 监控指标除以当前每秒执行的 SQL 语句数量。
 - Bytes Written：以 Resource Group 为单位进行实时统计的写入数据量。`total` 为当前所有 Resource Group 写入数据量之和。
 - Bytes Written Per Query：平均每个 SQL 语句的写入数据量。计算方法是将前述 Bytes Written 监控指标除以当前每秒执行的 SQL 语句数量。
 - KV CPU Time：以 Resource Group 为单位进行实时统计的 KV 层 CPU 时间消耗。`total` 为当前所有 Resource Group 消耗 KV 层 CPU 时间之和。
-- KV CPU Time Per Query：平均每个 SQL 语句的 KV 层 CPU 时间消耗之和。计算方法是将前述 KV CPU Time 监控指标除以当前每秒执行的 SQL 语句数量。
 - SQL CPU Time：以 Resource Group 为单位进行实时统计的 SQL 层 CPU 时间消耗。`total` 为当前所有 Resource Group 消耗 SQL 层 CPU 时间之和。
-- SQL CPU Time Per Query：平均每个 SQL 语句的 SQL 层 CPU 时间消耗之和。计算方法是将前述 SQL CPU Time 监控除以当前每秒执行的 SQL 语句数量。
+
+## Client 相关指标
```
Comment (hfxsd) [inline grafana-resource-control-dashboard.md:36]:
```suggestion
## Resource Controller Client 相关指标
```

File: grafana-resource-control-dashboard.md [hunk 3]
```diff
@@ -22,15 +22,23 @@ summary: 了解资源管控 (Resource Control) 的 Grafana Dashboard 中所展
...
 - SQL CPU Time：以 Resource Group 为单位进行实时统计的 SQL 层 CPU 时间消耗。`total` 为当前所有 Resource Group 消耗 SQL 层 CPU 时间之和。
-- SQL CPU Time Per Query：平均每个 SQL 语句的 SQL 层 CPU 时间消耗之和。计算方法是将前述 SQL CPU Time 监控除以当前每秒执行的 SQL 语句数量。
+
+## Resource Controller Client 相关指标
+
+- Active Resource groups：实时统计各个 Resource Controller Client 的 Resource Groups 数量。
+- Total KV Request Count：以 Resource Group 为单位，实时统计各个 Resource Controller Client 的 KV 请求数量。`total` 为 Resource Controller Client 下 KV 请求数量之和。
+- Failed KV Request Count：以 Resource Group 为单位，实时统计各个 Resource Controller Client 的 KV 失败请求数量。`total` 为 Resource Controller Client 下 KV 失败请求数量之和。
+- Seccessful KV Request Count：以 Resource Group 为单位，实时统计各个 Resource Controller Client 的 KV 成功请求数量。`total` 为 Resource Controller Client 下 KV 成功请求数量之和。
+- Seccessful KV Request Wait Duration (99/90)：以 Resource Group 为单位，实时统计各个 Resource Controller Client 成功 KV 请求等待时间（不同百分位）。
```
Comment (hfxsd) [inline grafana-resource-control-dashboard.md:42]:
```suggestion
- Successful KV Request Count：以 Resource Group 为单位，实时统计各个 Resource Controller Client 的 KV 成功请求数量。`total` 为 Resource Controller Client 下 KV 成功请求数量之和。
- Successful KV Request Wait Duration (99/90)：以 Resource Group 为单位，实时统计各个 Resource Controller Client 成功 KV 请求等待时间（不同百分位）。
```

File: grafana-resource-control-dashboard.md [hunk 4]
```diff
@@ -22,15 +22,23 @@ summary: 了解资源管控 (Resource Control) 的 Grafana Dashboard 中所展
...
+- Token Request Handle Duration (999/99)：以 Resource Group 为单位，实时统计各个 Resource Controller Client 向 Server 端申请 Token 等待响应时间（不同百分位）。
+- Token Request Count：以 Resource Group 为单位，实时统计各个 Resource Controller Client 向 Server 端申请 Token 次数。`seccessful` 和 `failed` 分别为 Resource Controller Client 下 KV 成功和失败请求数量之和。
```
Comment (hfxsd) [inline grafana-resource-control-dashboard.md:44]:
```suggestion
- Token Request Count：以 Resource Group 为单位，实时统计各个 Resource Controller Client 向 Server 端申请 Token 次数。`seccessful` 和 `failed` 分别为 Resource Controller Client 下 Token 请求成功和失败数量之和。
```

**Review by songrijie — APPROVED** [review]
LGTM

File: grafana-resource-control-dashboard.md [hunk 5]
```diff
@@ -22,15 +22,23 @@ summary: 了解资源管控 (Resource Control) 的 Grafana Dashboard 中所展
...
 - Bytes Read：以 Resource Group 为单位进行实时统计的读取数据量。`total` 为当前所有 Resource Group 读取数据量之和。
 - Bytes Read Per Query：平均每个 SQL 语句的读取数据量。将前述 Bytes Read 监控指标除以当前每秒执行的 SQL 语句数量。
 - Bytes Written：以 Resource Group 为单位进行实时统计的写入数据量。`total` 为当前所有 Resource Group 写入数据量之和。
 - Bytes Written Per Query：平均每个 SQL 语句的写入数据量。计算方法是将前述 Bytes Written 监控指标除以当前每秒执行的 SQL 语句数量。
 - KV CPU Time：以 Resource Group 为单位进行实时统计的 KV 层 CPU 时间消耗。`total` 为当前所有 Resource Group 消耗 KV 层 CPU 时间之和。
-- KV CPU Time Per Query：平均每个 SQL 语句的 KV 层 CPU 时间消耗之和。计算方法是将前述 KV CPU Time 监控指标除以当前每秒执行的 SQL 语句数量。
 - SQL CPU Time：以 Resource Group 为单位进行实时统计的 SQL 层 CPU 时间消耗。`total` 为当前所有 Resource Group 消耗 SQL 层 CPU 时间之和。
-- SQL CPU Time Per Query：平均每个 SQL 语句的 SQL 层 CPU 时间消耗之和。计算方法是将前述 SQL CPU Time 监控除以当前每秒执行的 SQL 语句数量。
+
+## Resource Controller Client 相关指标
```
Comment (qiancai) [inline grafana-resource-control-dashboard.md:36]: Resource Controller Client 是什么，这在资源管控的文档里没有提到过，是否需要在这里加一句描述

File: grafana-resource-control-dashboard.md [hunk 6]
```diff
@@ -22,15 +22,23 @@ summary: 了解资源管控 (Resource Control) 的 Grafana Dashboard 中所展
...
+
+- Active Resource groups：实时统计各个 Resource Controller Client 的 Resource Groups 数量。
```
Comment (qiancai) [inline grafana-resource-control-dashboard.md:38]:
```suggestion
- Active Resource Groups：实时统计各个 Resource Controller Client 的 Resource Groups 数量。
```

File: grafana-resource-control-dashboard.md [hunk 7]
```diff
@@ -22,15 +22,23 @@ summary: 了解资源管控 (Resource Control) 的 Grafana Dashboard 中所展
...
+- Total KV Request Count：以 Resource Group 为单位，实时统计各个 Resource Controller Client 的 KV 请求数量。`total` 为 Resource Controller Client 下 KV 请求数量之和。
+- Failed KV Request Count：以 Resource Group 为单位，实时统计各个 Resource Controller Client 的 KV 失败请求数量。`total` 为 Resource Controller Client 下 KV 失败请求数量之和。
+- Successful KV Request Count：以 Resource Group 为单位，实时统计各个 Resource Controller Client 的 KV 成功请求数量。`total` 为 Resource Controller Client 下 KV 成功请求数量之和。
+- Successful KV Request Wait Duration (99/90)：以 Resource Group 为单位，实时统计各个 Resource Controller Client 成功 KV 请求等待时间（不同百分位）。
```
Comment (qiancai) [inline grafana-resource-control-dashboard.md:42]:
```suggestion
- Successful KV Request Wait Duration (99/90)：以 Resource Group 为单位，实时统计各个 Resource Controller Client 成功 KV 请求的等待时间（不同百分位）。
```

File: grafana-resource-control-dashboard.md (hunk 5 above)
Comment (HuSharp) [reply grafana-resource-control-dashboard.md:38]: update. PTAL again, thx! @qiancai @Oreoxmt @CabinfeverB

Comment (qiancai) [reply grafana-resource-control-dashboard.md:38]: 具体是什么呢，下面提到了 “Active Resource Groups：实时统计各个 Resource Controller Client 的 Resource Groups 数量。”，说明  Resource Controller Client 是一个用于 xxx 的 Client?

Comment (HuSharp) [reply grafana-resource-control-dashboard.md:38]: 已更新，放在了此处解释。
https://github.com/pingcap/docs-cn/pull/13211/files#diff-afe90555b9f3fc8a48cf84963674fbe9117337b13d089abf69558e67341e05aaR14

File: grafana-resource-control-dashboard.md [hunk 8]
```diff
@@ -11,26 +11,36 @@ summary: 了解资源管控 (Resource Control) 的 Grafana Dashboard 中所展
 
 如果你的集群配置了 [Resource Control](/tidb-resource-control.md) ，通过观察 Resource Control 面板上的 Metrics，你可以了解当前集群整体的资源消耗状态。
 
+TiDB 使用[令牌桶算法](https://en.wikipedia.org/wiki/Token_bucket) 做流控，正如资源管控实现机制中所描述 [RFC: Global Resource Control in TiDB](https://github.com/pingcap/tidb/blob/master/docs/design/2022-11-25-global-resource-control.md#distributed-token-buckets)：一个 TiDB 节点可能存在多个 Resource Group（资源组），将在 PD 端的 GAC（Global Admission Control）进行流控。每个 TiDB 节点中的本地令牌桶（Local Token Buckets）将定期（默认 5 秒）与 PD 端的 GAC 进行通信，以重新配置本地令牌。其中的本地令牌桶（Local Token Buckets）具体实现为 Resource Controller Client。
```
Comment (qiancai) [inline grafana-resource-control-dashboard.md:14]:
```suggestion
TiDB 使用[令牌桶算法](https://en.wikipedia.org/wiki/Token_bucket) 做流控，正如资源管控实现机制 ([RFC: Global Resource Control in TiDB](https://github.com/pingcap/tidb/blob/master/docs/design/2022-11-25-global-resource-control.md#distributed-token-buckets)) 中所描述：一个 TiDB 节点可能存在多个 Resource Group（资源组），将在 PD 端的 GAC（Global Admission Control）进行流控。每个 TiDB 节点中的本地令牌桶（Local Token Buckets）将定期（默认 5 秒）与 PD 端的 GAC 进行通信，以重新配置本地令牌。其中的本地令牌桶（Local Token Buckets）具体实现为 Resource Controller Client。
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**